packages/zend-ldap/library/Zend/Ldap/Collection/Iterator/Default.php

```diff
@@ -312,7 +309,36 @@ public function rewind()
     #[ReturnTypeWillChange]
     public function valid()
     {
-        return (is_resource($this->_current));
+        return ($this->isResultEntry($this->_current));
     }
 
+    /**
+     * @param $resource
+     *
+     * @return bool
+     */
+    public function isResult($resource)
+    {
+        if(PHP_VERSION_ID < 80100)
```

Comment:
psr-2 formatting requires you put space between if and brance. this applies to all new code